src/cli.js

```diff
@@ -22,7 +22,7 @@ async function cli() {
     .parse(process.argv);
 
   let options = program.opts();
-  options = await parse(options);
+  options = await parse({...options, 'args': program.args.slice(1)});
```

Comment:
The added field `'args'` is not used, therefore this statement can be simplified:
```javascript
const options = await parse(program.opts());
```